bumble/device.py

```diff
@@ -1833,34 +1833,28 @@ def __init__(
 
     # [Classic only]
     @classmethod
-    def incomplete(cls, device, peer_address, role):
+    def from_pending_connection(
+        cls,
+        device: Device,
+        pending_connection: Device._PendingConnection,
+        handle: int,
+        parameters: Parameters,
+    ):
         """
-        Instantiate an incomplete connection (ie. one waiting for a HCI Connection
-        Complete event).
-        Once received it shall be completed using the `.complete` method.
+        Instantiate an connection.
```

Comment:
typo: an -> a
Maybe also be more descriptive. Like "Instantiate a Connection from a _PendingConnection and its now-resolved parameters"